Assets/GothicVR/Scenes/Bootstrap.unity

```diff
@@ -1258,6 +1259,102 @@ Transform:
   m_Children: []
   m_Father: {fileID: 1103036566}
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
+--- !u!82 &1283201087
+AudioSource:
```

Comment:
Why do we need an AudioSource on Bootstrap scene? Wouldn't it be better suited inside Menu.unity or General.unity?

Reply:
Before we already had the BackgroundMusic gameobject being available in the Bootstrap scene, but the audio source was created in script in MusicManager.
I just skipped the step in the script to create the Audio source, just so i can already set the output mixer, letting me control the volume perceived by the audio listener without directly controlling the volume of the audio source